### PR TITLE
Rdfa tags

### PR DESCRIPTION
**Pull request by felipeparaujo**

### CODE REVIEW

**Comment by programmes-developers** [comment]
Can one of the admins verify this patch?

**Comment by BjornTwachtmann** [comment]
Looks sensible to me.

File: src/ValueObject/MetaContext.php [hunk 1]
```diff
@@ -25,6 +29,12 @@ class MetaContext
     /** @var string */
     private $canonicalUrl = '';
 
+    /** @var string */
+    private $schemaType = '';
+
+    /** @var string */
+    private $vocabulary = 'http://schema.org/';
```
Comment (votemike) [inline src/ValueObject/MetaContext.php]: Does this need to be a property?

File: src/ValueObject/MetaContext.php [hunk 2]
```diff
@@ -83,6 +90,16 @@ public function titlePrefix(): string
         return $this->titlePrefix;
     }
 
+    public function getRdfaAttributes(): string
+    {
+        $tags = 'vocab=' . $this->vocabulary;
+        if (!empty($this->schemaType)) {
+            $tags .= ' typeof=' . $this->schemaType;
```
Comment (votemike) [inline src/ValueObject/MetaContext.php]: Tempted to build an array an implode on a space. But not super important.

File: src/ValueObject/MetaContext.php [hunk 3]
```diff
@@ -98,4 +115,27 @@ private function coreEntityTitlePrefix(CoreEntity $coreEntity): string
...
+
+        if ($entity instanceof Brand) {
+            return $type . 'Series';
+        }
+
+        if ($entity instanceof Clip) {
+            return $type . 'Clip';
+        }
+
+        return '';
```
Comment (votemike) [inline src/ValueObject/MetaContext.php]: throw an Exception???

File: src/ValueObject/MetaContext.php (hunk 1 above)
Comment (felipeparaujo) [inline src/ValueObject/MetaContext.php]: Not really, I just want to make clear which vocab we're using

File: src/ValueObject/MetaContext.php (hunk 3 above)
Comment (felipeparaujo) [inline src/ValueObject/MetaContext.php]: Why? it's just something we haven't mapped yet. In that case, we just don't add the typeof tag.

File: src/ValueObject/MetaContext.php (hunk 2 above)
Comment (felipeparaujo) [inline src/ValueObject/MetaContext.php]: I'm changing this, but it's going to look cleaner without the implode

File: src/ValueObject/MetaContext.php [hunk 4]
```diff
@@ -83,6 +87,15 @@ public function titlePrefix(): string
         return $this->titlePrefix;
     }
 
+    public function getRdfaAttributes(): string
+    {
+        if (empty($this->schemaType)) {
```
Comment (BPScott) [inline src/ValueObject/MetaContext.php]: `if (!$this->schemaType)`

**Pull request merged**